app/assets/stylesheets/splash.scss

```diff
@@ -26,19 +26,16 @@
     .welcome {
       height: 150px;
     }
-    .welcome a {
+    .welcome {
```

Comment:
Merge rule `.welcome` with rule on line 26<br>Selector should have depth of applicability no greater than 2, but was 3